Sources/CryptoService/DataTypes/JWKs.swift

```diff
@@ -0,0 +1,11 @@
+struct JWKs: Encodable {
+    let jwk: JWK
+}
+
+struct JWK: Encodable {
+    let kty = "EC"
+    let use = "sig"
+    let crv = "P-256"
+    let x: String
+    let y: String
```

Comment:
Might be worth making it super explicit what all of these values are:

```suggestion
    let keyType = "EC"
    let intendedUse: IntendedUse = .signing
    let ellipticCurve: EllipticCurve
    let x: String
    let y: String
    
    enum CodingKeys: String, CodingKey {
        case x, y
        case keyType = "kty"
        case intendedUse = "use"
        case ellipticCurve = "cry"
    }
    
    enum IntendedUse: String, Encodable {
        case signing = "sig"
    }
    
    enum EllipticCurve: String, Encodable {
        case primeField256Bit = "P-256"
    }
```

Useful docs: https://datatracker.ietf.org/doc/html/rfc7517#section-4.2
(possibly link these in a comment?)